### ISSUE TITLE
need hotkey to make tilda sticky

### ISSUE THREAD

**Issue by ri-aje** [lanoxx/tilda#324]
can we have a hotkey to quickly toggle tilda to make it sticky? by sticky, I mean don't disappear when losing focus, this allows tilda and other useful windows displayed at the same time to share information. there is the "hide when tilda loses focus" option that essentially does this, but adjusting it is much less convenient than pressing a hotkey. 

disappearing on focus loss works well most of the time, but occasionally it's much preferable to have tilda stay visible even when input focus is transferred to other windows. TotalTerminal for example has this ability. it's called "pin" in their setting.

**Comment by Kalabasa** [comment]
Isn't this the "Always on top" setting?

**Comment by ri-aje** [comment]
I don't see the "always on top" setting, are you talking about the "hide when tilda loses focus" setting?
in any event, I want a hotkey to quickly toggle this setting, instead of having to hunt it down in the config dialog. make sense?

**Comment by voidplayer** [comment]
general > window display >always on top

**Comment by ri-aje** [comment]
my overlook last time. 'always on top' is right there. though it doesn't do what I want, I want auto disappear most of the time, and stay visible on demand, such as by pressing a hot key. any idea/suggestion?

**Comment by voidplayer** [comment]
I understand your workflow. I dont think is currently possible with tilda. You want to toggle the autohide, not the always on top. Always on top is the same as "keep this window above others". Which my be easier to add a shortcut at system level, because is a feature of most window managers

Using a key to make it sticky sounds a lot like using a key to make it disappear and its only a key away ;)

I spend a lot of time in terminals so the autohide sounds very annoying to me

If you add the non-focus pull up behaviour to 'focus terminal' its even better imho, because you dont need to alt+tab into tilda, you can just press the hotkey to return to it. Making it possible to navigate between several tildas (I use 4 in 2 monitors) without using the mouse

Anyway, thats how I use it

Lannox is a debian user that took over the debian package and the upstream of this project like 3 years ago and im very thankful because he fixed some long standing issues and he comes and goes

We have a very long standing issue with not being able to get back to regular size after using full screen on a terminal, and hes been trying to make a new release with the latests fixes for 2 months now

What I mean by that is that I wouldnt hold my breath waiting for him to add new features soon :)

Because of that, anybody wanting to scratch their own itch is very welcome

**Comment by ri-aje** [comment]
yes, "hide when tilda loses focus" is the one I need a hotkey to toggle, given how tilda setting works today. I too stay in terminals as much as I can, though I use terminator. I found tilda is a really good all time available terminal for one off quick jobs and the same instance lives across workspaces, which is why I prefer it to autohide and stay visible only on demand. sounds like my ask is a no go for now.

that full screen bug is annoying as well, it literally locked my desktop and I have to resort to linux console to kill tilda (and all child processes) when that happens. I was about to file a bug for it, but I guess it's already on your list.
